bittensor/_dendrite/dendrite_impl.py

```diff
@@ -351,6 +351,114 @@ def _forward(
+            top_p (:obj: float, :default: 0.95): 
+                probability cutoff for top p sampling
+            temperature: (:obj: float, :default: 1.0):
+                The value used to module the next token probabilities for the softmax calculation
+            repetition_penalty (:obj: float, :default: 1.0):
+                The parameter for repetition penalty. 1.0 means no penalty.
+            length_penalty (:obj: float, :default: 1.0): 
+                The parameter for length penalty. 0.0 means no penalty, <0 to encourage longer sequences.
+            num_beam_groups (:obj: int, :default: 1):
+                Number of groups to divide num_beams into in order to ensure diversity among different groups of beams. 
+            max_time (:obj: float, :default: 150): 
+                The maximum time that a server can use to generate
+            forward_request_serializer_type (:obj:`bittensor.proto.Serializer.Type` of shape :obj:`(1)`, `optional`, :default: `bittensor.proto.Serializer.MSGPACK`):
+                Serializer used to pack torch tensors on forward request.
+            forward_response_serializer_type (:obj:`bittensor.proto.Serializer.Type` of shape :obj:`(1)`, `optional`, :default: `bittensor.proto.Serializer.MSGPACK`):
+                Serializer used to pack torch tensors on forward response.
+            backward_request_serializer_type (:obj:`bittensor.proto.Serializer.Type` of shape :obj:`(1)`, `optional`, :default: `bittensor.proto.Serializer.MSGPACK`):
+                Serializer used to pack torch tensors on forward request.
+            backward_response_serializer_type (:obj:`bittensor.proto.Serializer.Type` of shape :obj:`(1)`, `optional`, :default: `bittensor.proto.Serializer.MSGPACK`):
+                Serialzer used to pack torch tensors on backward response.
+        Returns:
+            generations (:obj:`List[str]`, `required`):
+                Generations from each endpoint.
+        """
+        tokenizer = bittensor.tokenizer()
+        response = self.text ( 
+            endpoints=endpoints, 
+            inputs = prompt,
+            synapses = [
+                synapse.TextSeq2Seq(
+                    topk = topk,
+                    num_to_generate = num_to_generate,
+                    num_beams = num_beams,
+                    no_repeat_ngram_size = no_repeat_ngram_size,
+                    early_stopping = early_stopping,
+                    num_return_sequences = num_return_sequences,
+                    do_sample = do_sample,
+                    top_p = top_p,
+                    temperature = temperature,
+                    repetition_penalty = repetition_penalty,
+                    length_penalty = length_penalty,
+                    max_time = max_time,
+                    num_beam_groups = num_beam_groups,
+                )
+            ],
+            timeout = timeout
+        )
+        generations = []
```

Comment:
`output, codes, times = response`
should we not verify no error codes before we attempt to decode the tokens.
Perhaps output a useful error message with the mapped error code message